src/layouts/base.njk

```diff
@@ -58,90 +58,93 @@
       </div>
     </section> 
         
-      <section id="projects" class="projects">
-        <h2>Projects</h2>
-        <div class="grid">
+      <section id="projects" class="cmp-projects">
+        <div class="cmp-projects__container">
+          <h2 class="cmp-projects__heading">Projects</h2>
 
-        <!-- neverland travelers -->
-        <div class="neverland-travelers">
-          <div class="left">
-            <div class="nt-text">
-              <h3>
-                Website for personal vlog
-              </h3>
-              <p>
-                I built and deployed the first version of a website dedicated to my personal vlog channel called Neverland Travelers. 
-              </p>
-              <div class="links">
-                <a href="work/neverland-travelers.html" class="link1">Learn More</a>
-                <a href="https://www.neverlandtravelers.com" target="_blank" class="link2">
-                  See Live Site &nbsp; &nbsp;  
-                  {% image 'src/public/usedresources/external-link.svg', '', 'Visit Neverland Travelers site' %}
-                </a>
+          <div class="obj-projects-grid">
+            {# neverland travelers vlog #}
+            <div class="obj-projects-grid__item obj-projects-grid__item--vlog">
+              {# text container on left #}
+              <div class="obj-projects-grid__item-text">
+                <h3 class="obj-projects-grid__heading">
+                  Website for personal vlog
+                </h3>
+                <p class="obj-projects-grid__description">
+                  I built and deployed the first version of a website dedicated to my personal vlog channel called Neverland Travelers. 
+                </p>
+                <div class="obj-projects-grid__links">
+                  <a href="/work/neverland-travelers.html" class="cmp-button__primary" aria-label="Learn more about the Neverland Travelers vlog project site">Learn More</a>
+                  <a href="https://www.neverlandtravelers.com" target="_blank" class="cmp-button__secondary">
```

Comment:
I'm planning on tackling all of the accessibility issues in a separate branch. Do you think the aria-label is more appropriate here, instead of the alt text?